yt_dlp/extractor/vidlii.py

```diff
@@ -85,7 +120,7 @@ def _real_extract(self, url):
         view_count = int_or_none(self._search_regex(
             (r'<strong>(\d+)</strong> views',
              r'Views\s*:\s*<strong>(\d+)</strong>'),
-            webpage, 'view count', fatal=False))
+            webpage.replace(",", ""), 'view count', fatal=False))
```

Comment:
why search-replace the whole webpage? Isnt this enough?
```suggestion
        view_count = str_to_int(self._search_regex(
            (r'<strong>([,0-9]+)</strong> views',
             r'Views\s*:\s*<strong>([,0-9]+)</strong>'),
            webpage, 'view count', fatal=False))
```